tests/sympc/tensor/mpc_tensor_test.py

```diff
@@ -653,3 +656,100 @@ def test_ops_different_share_class(get_clients) -> None:
 def test_get_shape_none() -> None:
     with pytest.raises(ValueError):
         MPCTensor._get_shape("mul", None, None)
+
+
+@pytest.mark.parametrize("bit", [0, 1])
+def test_bin_public_xor(get_clients, bit) -> None:
```

Comment:
I think the public xor test is repeated here.as I saw one test in `replicated_share_tensor_test.py`

Reply:
Ok, my mistake, this test is for session ring_size :+1:

Reply:
MAYBE: Could we move this test to `replicated_share_tensor_test` to maintain the consistency of having public xor rst like other ring_size(prime,binary)